portfolio/src/main/webapp/script.js

```diff
@@ -43,3 +43,25 @@ function addRandomGreeting() {
    const factContainer = document.getElementById('fact-container');
    factContainer.innerText = fact;
  }
+
+/*
+* Extends a collapsible element.
+*/
+function extend() {
+  var collapsibleElements = document.getElementsByClassName("collapsible");
+  var index;
+
+  //Add click listener to each collapsible element.
```

Comment:
This is pretty interesting! Couple of questions:

Does this mean that clicking on a different element will collapse the expanded element?

Also, we are adding these event listeners on every button click for "extend". Could we add the event listeners once the page loads?

Reply:
Clicking on the same element will collapse the expanded element. Clicking on other elements won't affect other expanded elements.

For the 2nd question, I tried replacing "click" with "load", but then the element never seems to expand. I am not really sure of another way to do this; do you have any suggestions? I will keep doing some research about it and try other things today.